docker-compose.local.yaml

```diff
@@ -27,5 +27,39 @@ services:
     volumes:
       - .:/app
 
+  redis:
+    container_name: care_redis
+    image: redis:alpine
+    restart: always
+    ports:
+      - "6379:6379"
+
+  celery:
+    container_name: care_celery
+    build:
+      context: .
+      dockerfile: docker/DevDockerfile
```

Comment:
This does not contain a beat instance, create another container for beat or add the beat functionality to this container.